### PR TITLE
Remove paramedic

### LINKED ISSUE #16034: Paramedic needs nerfs
1. It's PDA is better than an MD for no reason, the scan time is over twice as fast.
2. It shouldn't have its own roundstart hardsuit. That's what EVA is for.

### PR DESCRIPTION
**Pull request by AwareFoxy**
## About the PR
Removes paramedic role from the game.

## Why / Balance
Paramedic was a mistake, just a medic with different clothing and access to the medblock exit, station engs and secs can do this without a separate role.

closes #16034
paramedic clothing can be obtained in medical doctor loadout
## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- remove: Paramedic removed from the game.
- add: You can obtain paramedic clothing in medical doctor loadout.

### CODE REVIEW

**Comment by ArtisticRoomba** [comment]
Is this 2 year old referenced issue really still a problem today?

**Comment by lzk228** [comment]
> Is this 2 year old referenced issue really still a problem today?

that's what we are gonna figure out now

File: Resources/migration.yml [hunk 1]
```diff
@@ -440,3 +440,6 @@ BlueprintFlare: null
 
 # 2024-10-04
 BaseAdvancedPen: Pen
+
+# 2024-10-30
+LockerParamedic: null
```
Comment (AwareFoxy) [inline Resources/migration.yml:445]: remove final newline shit

**Comment by ArtisticRoomba** [comment]
Your build failed because you removed the paramedic job

**Comment by superjj18** [comment]
The only legitimate complaint in #16034 relates to the void suit as I think scan speed has been changed over the last two years especially with the pda scanner becoming an installable app.

This change adds nothing to the game, and with spacing being a common occurrence with asteroids being as they are there is zero reason why a doctor should have to go to EVA just to save a person.

Paramedics and Doctors are two vastly different roles irl, and that difference will only get larger once newmed eventually rolls out.

For those reasons I think this change is bad as it doesn't add anything nor does it really fix any true problems.  I've never once in my life found myself complaining that the paramedic of all things has a spacesuit.

**Comment by AwareFoxy** [comment]
> Your build failed because you removed the paramedic job
> 
> If you want this to work you shouldn't actually remove the paramedic roles, just remove the jobs on the stations and add the uniforms to loadout if you don't want to break anything



> Your build failed because you removed the paramedic job
> 
> If you want this to work you shouldn't actually remove the paramedic roles, just remove the jobs on the stations and add the uniforms to loadout if you don't want to break anything

yeah i fucked up, thank you
gonna do what ppl did with a brigmedic role

**Comment by ArtisticRoomba** [comment]
Like others have said, I would also like to point out that the paramedic and regular doctor are different roles with different responsibilities.

We don't really have a problem with the void suit IMO. I don't see or hear about it being abused.

**Review by VasilisThePikachu — COMMENTED** [review]
I don't have an opinion about whether to keep or remove peramed. But I do have opinions about not removing players medical playtime suddenly.

IMO before this gets merged #22418 needs to be addressed somehow.

**Comment by Savsj** [comment]
I have literally never seen the paramedic be considered a problem, like someone else said its a completely different job with different responsibilities we should not be removing paramedic

**Comment by thebadman4662** [comment]
> I don't have an opinion about whether to keep or remove peramed. But I do have opinions about not removing players medical playtime suddenly.
> 
> IMO before this gets merged #22418 needs to be addressed somehow.

Chemist role removed = boom back to intern. :trollface: 

Bet there would be quite few paras out there like that.

**Comment by a-person5660** [comment]
I dont think there is any major issue with paramed in the current version of SS14 and the void suit certainly isnt an issue. the issue you claim this "solves" is two years old and this PR doesnt even attempt to address the issue itself in any capacity and instead opts to remove the role entirely. i think its pretty self explanatory why this is a bad PR.

**Comment by keronshb** [comment]
I still stand by my comment that Paramed was a mistake in SS13 and 14. The access should've just been given to doctors and they should need to ask for the hardsuit.

This is for two reasons.
1) Reduces role bloat
2) Requires doctors to actually leave medical, instead of just making the paramed do it.

Many a SS13 round I've seen parameds just given the entire tools they need to fix someone fully out in the field, even invalidating the need for surgery and docs. This could either be done by on the fly surgery, chems, or both.

**Comment by SkippyMcSkips** [comment]
This is a terrible PR. If the issue is their PDA scan being too fast, then change that.

Paramedics are there to respond to people in distress across the station, stabilizing them and bringing them back to Medbay as quickly as they can where there are chems and such to better heal the crew members. 

Are you now expecting regular doctors to juggle people in the Medbay and across the station, and for those that take up the paramedic duties (if role is removed) to take the extra time to get permission to open EVA and also to get someone with access to actually open EVA to open it?

This just feels completely unnecessary and does not help the "SS14 devs love removing content for no good reason" sentiment.

**Comment by superjj18** [comment]
> I still stand by my comment that Paramed was a mistake in SS13 and 14. The access should've just been given to doctors and they should need to ask for the hardsuit.
> 
> This is for two reasons.
> 
> 1. Reduces role bloat
> 2. Requires doctors to actually leave medical, instead of just making the paramed do it.
> 
> Many a SS13 round I've seen parameds just given the entire tools they need to fix someone fully out in the field, even invalidating the need for surgery and docs. This could either be done by on the fly surgery, chems, or both.

I don’t think those two reasons are enough to justify its removal.  Referring to paramed as bloat seems very arbitrary, and I feel could be used to justify deleting nearly anything(ex: remove atmos, standard Engis can do their job, it’s bloat).

also nothing stops a doctor from leaving medical as is with the only exception being command members ordering otherwise, outside of that I’ve never seen a doctor get yelled at for not being in medbay(with the exception of the occasional AWOL chem who in all likelihood is probably missing because they rolled antag).  In SS13(at least in TG) field surgery typically needs knowledge of how to preform a particular surgery, whereas medbay has surgical computers that give step by step guides, meaning only experienced players can do it and even then it’s still time consuming compared to just dragging them to medbay.

If an injured person can be fully healed by a paramedic they likely weren’t that hurt to begin with.

**Comment by MozarteanChaos** [comment]
this _technically_ solves the issue, but... it's kind of the nuclear option, in my opinion. the first answer to "this needs to be nerfed" shouldn't be "well, let's remove it" unless that thing is severely broken... which paramedic isn't.

**Comment by a-person5660** [comment]
> this _technically_ solves the issue, but... it's kind of the nuclear option, in my opinion. the first answer to "this needs to be nerfed" shouldn't be "well, let's remove it" unless that thing is severely broken... which paramedic isn't.

this does not in any capacity solve the issue and we shouldnt even entertain the idea that this is an acceptable way to solve issues. this is how wizden gets a reputation for "har har lets remove fun pending rework™️".

**Comment by Bolper** [comment]
Хуйня какая-то

**Comment by Schrodinger71** [comment]
## Close please its pull pull request

**Comment by Bolper** [comment]
> ## Close please its pull request

I agree with you

**Comment by Schrodinger71** [comment]
Close that pull request bcs it's piece of bullshit

**Comment by lzk228** [comment]
as some maints will write, please don't be negative without a reason, if you have a complain, please describe it, otherwise your comments will be marked either as offtop or abuse

**Comment by BramvanZijp** [comment]
Guys, calm down. Its extremely unlikely thisll be merged, so let the maintainers handle it.

As for my opinion on the PR: There is NO reason to remove paramedic. Its fine balance wise, and if it wasnt, it should be nerfed not deleted.
Furthermore, why remove paramedic and not chemist, atmos, detective, etc.? These are all roles in in a department that specialize in something and are expected to be knowledgable in them. The paramedic is expected to recover bodies, chemists make chems, atmos sort atmospheric issues, detective investigate crimes, etc. But they can all still do the normal job of their department too. A paramedic or chemist can still help out in an overworked medbay, an atmostech can still repair damage, a detective can still go after a criminal, etc.

This PR singles out paramedic over an issue that was made 2 years ago and is completely out of date and no longer applies.
It no longer has better scan speed, the voidsuit is fine as is as it is fit for their job, engineers get hardsuits roundstart, should that be removed in favor of EVA? No. Paramedic is a fine role and has no reason to be removed.

**Comment by SkippyMcSkips** [comment]
> when I play doctor I carry chems in bottles on me and print out a defib for 3 steel at the medfab and do more than any paramed ever does
> 
> the paramed role itself teaches absolutely nothing new, barely introduces any new gameplay, and encourages people to do less of their job as medical staff...
> 
> anyone who wants the paramed playstyle (leave med to save people) can continue to do so if they want, not sure why it needs a specific role. same thing with how engis can choose to decide to go and fix structural issues or they can just sit in engi and babysit power.
> 
> I also don't like the void suit being seen as essential gear they need to have dibs on and wear all the time. Wearing the void suit all the time is the same bait as engis wearing their hardsuit all the time. More experienced players won't wear their suits unless it's absolutely needed because speed is everything in this game... and barely anyone dies in an area that's so spaced you need to be in the void suit for extended periods of time that you couldn't just survive with a basic mask and oxygen tank

"Why it needs a specific role"

The job of a paramedic and a regular Doctor in the game are very different. Paramedics should be able to fill in as a doctor in an emergency, but paramedic is primarily there to stabilize and bring hurt people to Medical.

As someone else said, you can make the exact same argument about Atmos techs, considering they're just Engis by another name. 

How does it encourage people to do less of their job as medical staff? The medical staffs job is to heal people. Paramedics stabilize and bring people to Medical when they're otherwise unable to themselves, which as it happens is pretty often at least in my experience. 

I also don't understand the issue with the void suit. It's not "essential" gear, but it can be very helpful for Paramedics or someone fulfilling the Paramedic duties (role or not). Some tiny extra protection, not having to rely on someone being able and available to open EVA for you, etc. Even when worn all shift, I don't understand the issue with it. It's not super powerful to where wearing it makes you notably stronger than anyone else, it's just minor protection. If people are wearing it unnecessarily, again, it's only hurting them by slowing the wearer down, which is not a game issue that's a player knowledge issue.

If there are balancing issues with the Void Suit and Paramedic PDA, those should be looked into an PR'd without removing an entire job role.

**Comment by Darkiich** [comment]
This is a cringe

**Comment by Krovotushka** [comment]
Based PR, concept of paramedic on space station sounds unrealistic cause of station's size in general, its like having a whole paramedic team in prison. Also most of the times parameds just do nothing..? They dont even heal people at place where they dropped in crit(from what i saw)

**Comment by Simyon264** [comment]
The argument that "station engs and secs can do this without a separate role." is not applicable as saving bodies from space is not the job of either of those roles. Secs hardsuits are also locked behind the armory, requiring either the HoS or Warden.

**Comment by SaphireLattice** [comment]
I can see the paramedic role being somewhat of an "RP extra", though it does give players a nudge to stay out and about without having to realize they can do this. Medical doctors are expected to stay in medbay and realizing you can just go out and patrol around the station is… not always easy. Though I suppose paramedic role existing tamps down on that too, but yeah. I think it gives people a nice nudge. 

I suppose it’s an issue of the role being basically a subtype of general medbay worker, with there being no system for this other than full jobs. Loadouts sorta fix this but not quite, as they are not exactly prominent and discoverable.

Another issue is that current medical system allows for full treatment to 0 damage in the field. This feels counter to this entire separation, and is likely something that will be felt much sharper when newmed comes out.

Round start hardsuits… who the hell has time to walk medical to EVA while the medbay is likely actively flooded with people because of whatever *caused* the spacing? And emergency suits don’t have a great availability, especially for medbay roles who will stay at the post for way too late. I am not sure I see a problem with them being available to medical overall. Maybe I’m missing something?

As for this PR from technical standpoint:
- The thief objective of a paramed voidsuit is kept, yet the lockers are nuked, unless I misread it? This feels like two different approaches to removal being done at once. Also cuts out voidsuits available to medical doctors.
- Role count is not adjusted. This PR effectively cuts out several job slots entirely without bumping doctor count to compensate.
- Paramedic lockers being deleted might cause some maps to look rather sparse in areas where they have been. Check for that too.

**Comment by thebadman4662** [comment]
> Though wonder if doctors would stop staying in medbay 24/7 if this is removed?

Doctors would first need to stop being allergic to bottles... there being only 12 bottles(epi bottles can be repurposed but you do not want to spill the chems, duh!) and only behind chem access on most maps does not help. Not sure if https://github.com/space-wizards/space-station-14/pull/33018 is in effect for medbay or we need to wait few weeks still but till it is medbays need a lot of glass before doctors feel more comfortable with patrols as people actually using bottles are said parameds.

**Comment by Erisfiregamer1** [comment]
My personal opinion: Removing the Paramed role is just going to make the game that bit worse and leave many more unfortunate people rotting in the maintenance shafts, unless Medical makes a replacement.

Having a dedicated role is nice, and the community clearly seems to agree:
image

At least put it to poll before this is merged.

**Comment by cohanna** [comment]
An aspect of gameplay people haven't brought up yet is that because the paramed specializes in body recovery people tend to allow them into their departments the same as a jani. If the role were to be removed people would have to allow some random doctor in who is fulfilling the role of the paramedic. This both reduces the likelihood of doctors filling this role (b/c people will be apprehensive to letting them in) and if they are let in and commit a crime it will be harder to narrow down the perpetrator due to the multiple amount of doctors. 
Essentially the paramed fills a role the same as atmos does. Sure an engine can fix atmospheric issues, the same a doctor can recover a body (i do it all the time for both roles) but the specialized roles exist to both ensure those jobs get done and to add more dynamic to gameplay. Gutting it solves a nonexistent issue (unless the maintainers are way out of wack from the playerbase.... again....) and just serves to make the game blander.

**Comment by LittleNorthStar** [comment]
The Paramed role is a critical safety net to stop people being round removed, both due to the paramedics ability to go to most areas and their vigilance on the crew monitor, additionally they are often the most active in making sure people have suit sensors on which can save a lot of newer players or forgetful veterans

I believe their removal will have nothing but a negative impact on the game and its balance I cannot see a single positive in doing so

**Comment by vorkathbruh** [comment]
i personally never saw why paramedic needed to be a discrete role

**Comment by SaphireLattice** [comment]
From mechanical standpoint paramedics do not have any difference from medical doctors beyond colors and a spawn time rollerbed. I'm not sure they even have any difference in access? They had external access nuked ages ago already. There's... nothing stopping you from playing a medical doctor like a paramedic beyond at best cultural things.

Removing it probably won't have as much of an impact but the role's existence gives a rather decent *guidance* to the player of what they are supposed to do.

Though again, for all I know removing this will change nothing and the people will adjust to have a dedicated person or a rotation of them doing the same thing, but as a medical doctor. Currently doing that would require there being no paramedics in first place, and pushing against people saying "But I'm not a paramedic!".

Personally the only issue I have with this PR is the voidsuit removal - that thing's paper thin and only protects from spacing. And space protection is extremely lacking in medbays, especially when the station becomes compromised and with every new person coming in, more air gets lost every minute. And of course any emergency suit nearby is already gone...

**Comment by Plykiya** [comment]
One fact that people also seemed to agree with when I brought it up is that medical's radio channel is basically completely unused during rounds. Current radio usage is limited to people yelling about coordinates, or complaining that you're out of one chemical fifty minutes into the shift.

I think that the paramedic role has slightly to do with that fact. When people need healing in an anrea, they'll say "paramed to x" on the radio. They're specifically asking for the one or two parameds. There's no coordination required in the medical channel and it is left completely in the air whether the paramed actually went there or not. The paramed essentially separates themselves from the medical department to work solo and does not properly promote radio RP.

Compare it to the radio usage of security or engineering. If someone calls security or engineering to a location, people will generally try to coordinate among themselves with who does what, who's going where, and whether a problem is solved or not. There is much more accountability.

At this current point in time paramedic is an antag roller's dream since no one ever checks up on them and no one ever confirms that they have retrieved anyone they say they did. If more medical doctors choose to respond to call outs for "dead body at x" or "med to x", this was naturally encourage more RP among the radio channel so that people aren't showing up to the same place and there will be actual coordination required over the radio.

**Comment by superjj18** [comment]
> I also don't like the void suit being seen as essential gear they need to have dibs on and wear all the time. Wearing the void suit all the time is the same bait as engis wearing their hardsuit all the time. More experienced players won't wear their suits unless it's absolutely needed because speed is everything in this game... and barely anyone dies in an area that's so spaced you need to be in the void suit for extended periods of time that you couldn't just survive with a basic mask and oxygen tank

If you don't wear the engi hardsuit 95% of the time I'm going to consider you absolutely uselsss and probably cuss you out in character for being a stupid dumbass who needs to walk across the station to get a suit every time space dust hits a window(IE every 3 minutes)

EVA is essential and the fear around giving it out to essential jobs is cringe.

**Comment by Jezithyr** [comment]
Paramed is *definitely* is 100% needed for newmed so *even if*  we ended up merging this (which does not look very likely), it would only be a temporary removal at worst. I completely understand where Keron is coming from though, paramed in 13 and how it is now in 14 is a redundant flavour role that is better off as an alternative job title for doctor rather than it's own role (That way there are more doctor slots and people can *choose* if they want to be "paramed"). 

That said, one of the major pillars of newmed's design is urgent care and triage. This makes having a separate paramedic role (and possibly multiple of them) a hard requirement due to the different skillset required. Paramedics will need to rapidly get to, diagnose and quickly *stabilize* patents so that they can survive the trip to medical. The goal of a paramed is to do everything possible to make sure that the patient can get to medical, *even if it means causing permanent damage* or doing things "improperly". Parameds do not *treat* injuries, they hold you together with spit and ducktape before dumping you onto the medical bed for a doctor to actually fix. (And in many cases this will mean *fixing the damage* that came from what the paramed had to do to keep the patient alive.

**Comment by UncaughtEx** [comment]
I'd just like to add as a side comment that the psychologist as a role can be accused as a form of role bloat and currently doesn't fulfill any purpose outside of RP due to dramatic changes to space-law much in the same way the paramed does in this PR's context.

**Comment by Vonsant** [comment]
image

**Comment by UncaughtEx** [comment]
> image

Peak Image but in response to the person developing it is even more hilarious

**Comment by thebadman4662** [comment]
> > I also don't like the void suit being seen as essential gear they need to have dibs on and wear all the time. Wearing the void suit all the time is the same bait as engis wearing their hardsuit all the time. More experienced players won't wear their suits unless it's absolutely needed because speed is everything in this game... and barely anyone dies in an area that's so spaced you need to be in the void suit for extended periods of time that you couldn't just survive with a basic mask and oxygen tank
> 
> If you don't wear the engi hardsuit 95% of the time I'm going to consider you absolutely uselsss and probably cuss you out in character for being a stupid dumbass who needs to walk across the station to get a suit every time space dust hits a window(IE every 3 minutes)
> 
> EVA is essential and the fear around giving it out to essential jobs is cringe.

But how often engi actually need hardsuit vs EEVA that works just fine if you do not try to spacewalk?

Also spacing sounds like atmos issue which have 4x4 softsuit anyways. :trollface:

**Comment by cohanna** [comment]
> If you don't wear the engi hardsuit 95% of the time I'm going to consider you absolutely uselsss and probably cuss you out in character for being a stupid dumbass who needs to walk across the station to get a suit every time space dust hits a window(IE every 3 minutes)

Most of the highest playtime engies and atmos don't use their suits. 
As an engie main, spacing damage is so little you legit don't need a hardsuit unless its like syndiebomb level damage or station wide spacing.
Its really just a safety net for newer engies.  You don't even need it to do solars, as long as you have a winter coat (i don't even bother turning on o2 unless im gonna be in a spaced area for like 2+ mins).
Anyone who is running back to throw on their hardsuit for a couple windows or walls is a new engie.

**Comment by NoElkaTheGod** [comment]
Have you even seen that many dislikes on a PR? I for sure did not. I think this might mean something.
Other than that, other people have already said everything there is. There aren't really any isues with paramedic, hence no real reason to remove it.

**Comment by VasilisThePikachu** [comment]
After an internal chat with multiple maintainers, we concluded that we would like to keep paramedic into the game. Some reasons include
- Peramed is important to newmed/woundmed. Not worth a temporary removal.
- Peramed is believed to have enough unique gameplay compared to medical doctors.
- #16034 should be solved.
- The reasoning in the pr itself is not very good.

I would also like to say I am VERY disappointed with some people's reactions here. The PR creator was just trying to find issues to close and just had a different opinion. However, people flooded the comments to post stupid memes, send threats to the creator or in general make comments that do not contribute to the discussion.

I bet you would not want to be on the receiving side either, I have and it's NOT fun. Please try to keep discussions on GitHub civil and friendly. I will be requesting repo bans to be issued next time I see shitposting and harassment on GitHub.

I will also be locking any further discussion in this PR.

**Pull request closed without merging**